src/deluge/gui/ui_timer_manager.cpp

```diff
@@ -129,9 +129,8 @@ void UITimerManager::routine() {
 				}
 
```

Comment:
code cleanup throughout to call isOnAutomationOverview instead of doing a direct check comparing lastSelectedParamID to 255.